lib/gcp-compute-engine.go

```diff
@@ -213,6 +204,15 @@ func getInstanceName() string {
 	return strings.Split(hostName, ".")[0]
 }
 
+func getGoogleClient(ctx context.Context, apiKey string) (*http.Client, error) {
+	if apiKey == "" {
+		return google.DefaultClient(ctx, monitoring.MonitoringReadScope)
```

Comment:
ref: https://cloud.google.com/monitoring/api/authentication

Reply:
[memo]
- MonitoringReadScope is smaller scope, and it's enough for this monitoring plugin